src/components/primaryButton/PrimaryButton.svelte

```diff
@@ -0,0 +1,47 @@
+<script lang="ts">
+    import type { PrimaryButtonInputs } from '../../lib/types/primaryButton/PrimaryButton';
+    import Spinner from '../../lib/images/Spinner.svelte';
+    import CheckMark from '../../lib/images/Vector.svelte';
+    export let primaryButtonInputs: PrimaryButtonInputs;
+    let isLoading = false;
+    let isSuccess = false;
+
+    const { buttonText, clickAction, isDisabled, successMessage } = primaryButtonInputs;
+    const defaultStyle =
```

Comment:
Can we combine the common classes to make it easier to see what's changing between the two?

Reply:
We can but the downside is its ugly concatenating the strings I had initially tried to only have one class that would just swap out the color but it ended up looking gross so I just went with two separate classes